snaps/clean.go

```diff
@@ -106,7 +106,7 @@ func Clean(m *testing.M, opts ...CleanOpts) {
 		testEvents.items,
 		shouldClean && !isCI,
 	); s != "" {
-		fmt.Print(s)
+		fmt.Printf("%s\n", s)
```

Comment:
trying with `fmt.Println` also solves this issue. Not sure why it behaves like this. I am guessing it has to do something with go test and the verbose flag capturing output

Reply:
Should we use `fmt.Println` instead?

Reply:
oh yeah duh of course you're right and that makes more sense - for some very silly reason I was mixing up `Print` with `Println`.

```suggestion
		fmt.Println(s)
```